### PR TITLE
net-dialup/freeradius: Fix dependency issues

### PR DESCRIPTION
**Pull request by herbmillerjr**
PR for a couple of open bugs against package.

### CODE REVIEW

**Comment by herbmillerjr** [comment]
I kept 3.0.17 as is to make the changes I made clearer in the diff, but should I revbump this for dependency bugs?

**Comment by herbmillerjr** [comment]
I opened this one a little prematurely. I want to knock out the samba bug (https://bugs.gentoo.org/638396) as well.

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 1]
```diff
@@ -34,10 +34,12 @@ RDEPEND="!net-dialup/cistronradius
 	python? ( ${PYTHON_DEPS} )
 	readline? ( sys-libs/readline:0= )
 	pcap? ( net-libs/libpcap )
+	memcached? ( dev-libs/libmemcached )
 	mysql? ( virtual/mysql )
 	postgres? ( dev-db/postgresql:= )
 	firebird? ( dev-db/firebird )
 	pam? ( virtual/pam )
+	rest? ( dev-libs/json-c )
```
Comment (mgorny) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: This one needs `:=`.

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 2]
```diff
@@ -79,7 +80,8 @@ src_prepare() {
 	use pam || { rm -r src/modules/rlm_pam || die ; }
 	use python || { rm -r src/modules/rlm_python || die ; }
 	use rest || { rm -r src/modules/rlm_rest || die ; }
-	# Do not install ruby rlm module, bug #483108
```
Comment (mgorny) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: Any specific reason to nuke this comment?

File: net-dialup/freeradius/metadata.xml [hunk 3]
```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!-- maintainer-needed -->
+  <maintainer type="person">
```
Comment (mgorny) [inline net-dialup/freeradius/metadata.xml]: Looks like you've put a lot of unrelated changes in this commit. Please split.

File: net-dialup/freeradius/freeradius-3.0.17.ebuild (hunk 2 above)
Comment (mgorny) [reply net-dialup/freeradius/freeradius-3.0.17.ebuild]: Still applies.

Comment (herbmillerjr) [reply net-dialup/freeradius/freeradius-3.0.17.ebuild]: Nope, didn't realize I did.

**Comment by mgorny** [comment]
Thanks. Looks good now. I'll just test and merge.

**Comment by mgorny** [comment]
Looks like @pacho2 fixed the bindist USE-dep already. Merged the rest as 8b1154ee1088d4a6d400e21d54036e396c227c5a, b955066a2726c734664fe8d000c4b05675557707, e009aad446d86af40332a3d22f0c716838407472.
